mods/d2k/rules/world.yaml

```diff
@@ -35,6 +35,10 @@
 		Name: Corrino
 		InternalName: corrino
 		Selectable: false
+	Faction@Smuggler:
```

Comment:
This doesn't appear to be used yet, but if you want to keep it will need entries set up in the building `FactionImages` dictionaries.